app/src/main/java/app/revanced/integrations/youtube/shared/VersionCheck.java

```diff
@@ -0,0 +1,10 @@
+package app.revanced.integrations.youtube.shared;
+
+import app.revanced.integrations.shared.Utils;
+
+public class VersionCheck {
```

Comment:
Should this be called `VersionCheckPatch` for consistency like the other patches?